alpine-rails-nginx/Dockerfile

```diff
@@ -31,10 +31,10 @@ COPY nginx-remove-server-name.patch /tmp/
 RUN /tmp/install-nginx.sh \
   && rm -rf -- /usr/local/src /tmp/*
 
+# Setup bundler for the root user
+RUN bundle config --global -- gemfile '/opt/rails/Gemfile' \
+  && bundle config --global -- deployment
```

Comment:
You probably meant `--deployment` not `-- deployment` (not the whitespace difference).